src/databricks/labs/remorph/config.py

```diff
@@ -8,8 +8,11 @@
 
 @dataclass
 class MorphConfig:
+    __file__ = "config.yml"
+    __version__ = 2
+
     source: str
-    input_sql: str
+    input_sql: str | None
```

Comment:
in Hindsight let us capture defaults for all variables,  how are we testing cli is picking up defaults from the user folder and allows local overrides?